text/0097-log-data-model.md

```diff
@@ -0,0 +1,1005 @@
+  </tr>
+  <tr>
+    <td>eventSource</td>
+    <td>string</td>
+    <td>The service that the request was made to. This name is typically a short form of the service name without spaces plus .amazonaws.com.</td>
+    <td>Resource["service.name"]?</td>
+  </tr>
+  <tr>
+    <td>awsRegion</td>
+    <td>string</td>
+    <td>The AWS region that the request was made to, such as us-east-2.</td>
+    <td>Resource["cloud.region"]</td>
+  </tr>
+  <tr>
+    <td>sourceIPAddress</td>
+    <td>string</td>
+    <td>The IP address that the request was made from.</td>
+    <td>Resource["net.peer.ip"] or Resource["net.host.ip"]? TBD</td>
+  </tr>
+  <tr>
+    <td>errorCode</td>
+    <td>string</td>
+    <td>The AWS service error if the request returns an error.</td>
+    <td>ShortName</td>
+  </tr>
+  <tr>
+    <td>errorMessage</td>
+    <td>string</td>
+    <td>If the request returns an error, the description of the error.</td>
+    <td>Body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>*</td>
+    <td></td>
+    <td>Attributes["cloudtrail.*"]</td>
+  </tr>
+</table>
+
+### Google Cloud Logging
+
+Field            | Type               | Description                                             | Maps to Unified Model Field
+-----------------|--------------------| ------------------------------------------------------- | ---------------------------
+timestamp        | string             | The time the event described by the log entry occurred. | Timestamp
+resource         | MonitoredResource  | The monitored resource that produced this log entry.    | Resource
+log_name         | string             | The URL-encoded LOG_ID suffix of the log_name field identifies which log stream this entry belongs to. | ShortName
+json_payload     | google.protobuf.Struct | The log entry payload, represented as a structure that is expressed as a JSON object. | Body
+proto_payload    | google.protobuf.Any | The log entry payload, represented as a protocol buffer. | Body
+text_payload     | string             | The log entry payload, represented as a Unicode string (UTF-8). | Body
+severity         | LogSeverity        | The severity of the log entry.                          | Severity
+trace            | string             | The trace associated with the log entry, if any.        | TraceId
+span_id          | string             | The span ID within the trace associated with the log entry. | SpanId
+labels           | map<string,string> | A set of user-defined (key, value) data that provides additional information about the log entry. | Attributes
+All other fields |                    |                                                         | Attributes["google.*"]
+
+## Appendix B: `SeverityNumber` example mappings
+
+|Syslog       |WinEvtLog  |Log4j |Zap   |java.util.logging|SeverityNumber|
+|-------------|-----------|------|------|-----------------|--------------|
+|             |           |TRACE |      | TRACE           |TRACE         |
```

Comment:
According to https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Level.html, `java.util.logging` doesn't have `TRACE` level.  The `TRACE` should probably be replaced by `FINEST`.

Reply:
Fixed.